data-sources/flutter-web/config.yml

```diff
@@ -0,0 +1,14 @@
+id: flutter-web
+displayName: Flutter Web
+description: |
+  Monitor your Flutter applications, and analyze performance bottlenecks while operating in a live development environment. Visually monitor your web user experience across the world with your application rendition via the dashboard.
```

Comment:
> Visually monitor your web user experience across the world with your application rendition via the dashboard.

What do you mean by this? The word rendition seems a bit out of context for this sentence.

Reply:
thank you @brammerl , I have fixed this typo and updated this PR 👍